test/Parse/line-directive.swift

```diff
@@ -1,4 +1,5 @@
 // RUN: %target-typecheck-verify-swift
+// RUN: (%target-swift-frontend -c %s 2>&1 || echo -n) | %FileCheck %s
```

Comment:
Since you expect it to fail, you can use the `not` utility (which comes with LLVM), i.e. `not %target-swift-frontend -typecheck %s 2>&1 | %FileCheck %s`.

Reply:
Done. Ta.